### PR TITLE
Update instructions to prepare backport packages

### PR DESCRIPTION
**Pull request by kaxil**
Update instructions to prepare backport packages

---

Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -998,10 +998,12 @@ prepare such packages on your own easily.
 
 * Enter the ``backport_packages`` directory
 
+* Run ``python setup_backport_packages.py prepare`` to update deprecated modules & paths.
+
 * Usually you only build some of the providers package. The ``providers`` directory is separated into
   separate providers. You can see the list of all available providers by running
   ``python setup_backport_packages.py list-backport-packages``. You can build the backport package
-  by running ``python setup.py <PROVIDER_NAME> bdist_wheel``. Note that there
+  by running ``python setup_backport_packages.py <PROVIDER_NAME> bdist_wheel``. Note that there
```
Comment (turbaszek) [inline CONTRIBUTING.rst]: I think we should encourage people to use one-step-only:
```
scripts/ci/ci_prepare_packages.sh packageA packageB packageC
```
WDYT?

Comment (potiuk) [reply CONTRIBUTING.rst]: Yes. We should do it via script as it additionally cleans up whatever needs to be cleaned up. I've had a lot of troubles in the past trying to build package and forgot about cleaning.

Comment (kaxil) [reply CONTRIBUTING.rst]: Fixed

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -996,18 +996,22 @@ prepare such packages on your own easily.
 * Make sure you have ``setuptools`` and ``wheel`` installed in your python environment. The easiest way
   to do it is to run ``pip install setuptools wheel``
 
-* Enter the ``backport_packages`` directory
+* Run the following command:
+
+  .. code-blamazonock:: bash
```
Comment (potiuk) [inline CONTRIBUTING.rst]: @kaxil - I am afraid you just revealed your super secret password :)

Comment (kaxil) [reply CONTRIBUTING.rst]: haha. lol , have no idea what it is. 

oh wait, yeah I was trying `amazon` backport and copy pasted it between ``..code-block`` :D

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -996,18 +996,22 @@ prepare such packages on your own easily.
...
   separate providers. You can see the list of all available providers by running
-  ``python setup_backport_packages.py list-backport-packages``. You can build the backport package
-  by running ``python setup.py <PROVIDER_NAME> bdist_wheel``. Note that there
+  ``./scripts/ci/ci_prepare_packages.sh --help``. You can build the backport package
+  by running ``./scripts/ci/ci_prepare_packages.sh <PROVIDER_NAME>``. Note that there
   might be (and are) dependencies between some packages that might prevent subset of the packages
   to be used without installing the packages they depend on. This will be solved soon by
   adding cross-dependencies between packages.
 
 * You can build 'all providers' package by running
```
Comment (potiuk) [inline CONTRIBUTING.rst]:
```suggestion
```

File: CONTRIBUTING.rst [hunk 4]
```diff
@@ -996,18 +996,22 @@ prepare such packages on your own easily.
...
-  ``python setup_backport_packages.py providers bdist_wheel``. This package contains all providers thus
+  ``./scripts/ci/ci_prepare_packages.sh``. This package contains all providers thus
```
Comment (potiuk) [inline CONTRIBUTING.rst]:
```suggestion
```

File: CONTRIBUTING.rst [hunk 5]
```diff
@@ -996,18 +996,22 @@ prepare such packages on your own easily.
...
   it does not have issues with cross-dependencies.
```
Comment (potiuk) [inline CONTRIBUTING.rst]:
```suggestion
```

Comment (kaxil) [reply CONTRIBUTING.rst]: fixed

File: CONTRIBUTING.rst (hunk 4 above)
Comment (kaxil) [reply CONTRIBUTING.rst]: fixed

File: CONTRIBUTING.rst (hunk 3 above)
Comment (kaxil) [reply CONTRIBUTING.rst]: fixed

**Review by turbaszek — APPROVED** [review]

**Pull request merged**